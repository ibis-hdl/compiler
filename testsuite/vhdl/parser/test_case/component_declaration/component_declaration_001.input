component INV is
    port (A: in  STD_LOGIC;
          F: out STD_LOGIC
    );
end component;

