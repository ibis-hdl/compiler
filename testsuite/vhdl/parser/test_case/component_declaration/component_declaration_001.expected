(component_declaration
  (identifier
    INV
  )
  (port_clause
    (interface_list
      (interface_declaration<v>
        (interface_constant_declaration
          (identifier_list
            (identifier
              A
            )
          )
          (in)
          (subtype_indication
            (type_mark<v>
              (name<v>
                (identifier
                  STD_LOGIC
                )
              )
            )
          )
        )
      ),
      (interface_declaration<v>
        (interface_signal_declaration
          (identifier_list
            (identifier
              F
            )
          )
          (keyword
            OUT
          )
          (subtype_indication
            (type_mark<v>
              (name<v>
                (identifier
                  STD_LOGIC
                )
              )
            )
          )
        )
      )
    )
  )
)
